vmdb/app/models/ems_container.rb

```diff
@@ -3,13 +3,24 @@ class EmsContainer < ExtManagementSystem
     EmsKubernetes
   )
 
+  has_many :container_nodes,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_groups,   :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_services, :foreign_key => :ems_id, :dependent => :destroy
+
   def self.supported_subclasses
     subclasses
   end
 
-  # Preload any subclasses of this class, so that they will be part of the
-  #   conditions that are generated on queries against this class.
-  EmsContainer::SUBCLASSES.each { |c| require_dependency Rails.root.join("app", "models", "#{c.underscore}.rb").to_s }
```

Comment:
Yes it was here twice. For a better diff look at https://github.com/simon3z/manageiq/commit/f99c106d09c78b24fb7e502deca6dcba2f2ca623